### PR TITLE
Add company icon to login screen in Dashboard

### PR DESCRIPTION
**Pull request by muspelheim**
I added the company logo on the login page. because it is much more attractive than text

### CODE REVIEW

**Comment by OskarStark** [comment]
Can you add a screenshot @muspelheim 

I like your PR :+1:

**Comment by muspelheim** [comment]
[Click here](http://pix.toile-libre.org/upload/original/1459781816.png)

Looks pretty good)

**Comment by muspelheim** [comment]
selection_003

**Comment by OskarStark** [comment]
nice!

File: Resources/views/Admin/Security/login.html.twig [hunk 1]
```diff
@@ -16,8 +16,13 @@
...
             <a href="{{ path('sonata_admin_dashboard') }}">
-                {% if 'single_text' == admin_pool.getOption('title_mode') or 'both' == admin_pool.getOption('title_mode') %}
-                    <span>{{ admin_pool.title }}</span>
+                {% if 'single_image' == sonata_admin.adminPool.getOption('title_mode') or 'both' == sonata_admin.adminPool.getOption('title_mode') %}
+                    <div>
+                        <img style="width:64px;" src="{{ asset(sonata_admin.adminPool.titlelogo) }}" alt="{{ sonata_admin.adminPool.title }}">
+                    </div>
+                {% endif %}
+                {% if 'single_text' == sonata_admin.adminPool.getOption('title_mode') or 'both' == sonata_admin.adminPool.getOption('title_mode') %}
+                    <span>{{ sonata_admin.adminPool.title }}</span>
```
Comment (greg0ire) [inline Resources/views/Admin/Security/login.html.twig]: These lines are too long.

Comment (greg0ire) [inline Resources/views/Admin/Security/login.html.twig]: Argh too late.

Comment (muspelheim) [inline Resources/views/Admin/Security/login.html.twig]: i can add subtask for this issue, if you want

Comment (greg0ire) [inline Resources/views/Admin/Security/login.html.twig]: Do as you wish, it's not a big issue, can be fixed next time.

**Comment by greg0ire** [comment]
@muspelheim : @ejkun and I are wondering why you changed `admin_pool` to `sonata_admin.adminPool`. Can you please explain?

**Pull request merged**